README.md

```diff
@@ -320,6 +320,17 @@ Racecar supports [Datadog](https://www.datadoghq.com/) monitoring integration. I
 * `datadog_namespace` – The namespace to use for Datadog metrics.
 * `datadog_tags` – Tags that should always be set on Datadog metrics.
 
+#### Prometheus monitoring
+
+Racecar supports [Prometheus](https://www.prometheus.io/) monitoring integration. If you are scraping data from other applications, you just need to set `prometheus_enabled` to `true` and have `prometheus-client` gem on your dependencies, as the rest of the settings come with sane defaults.
+
+* `prometheus_enabled` – Whether prometheus monitoring is enabled (defaults to `false`) or not.
+* `prometheus_endpoint` – The server metrics endpoint that is used to fetch prometheus data.
+* `prometheus_registry` – The prometheus client instance.
```

Comment:
```suggestion
* `prometheus_registry` – The Prometheus client instance.
```

Can you link to some docs for this and perhaps the other concepts? Like, is this a URL for a service or something?